internal/provider/resource_consul_cluster.go

```diff
@@ -129,6 +129,12 @@ func resourceConsulCluster() *schema.Resource {
 					return strings.ToLower(old) == strings.ToLower(new)
 				},
 			},
+			"auto_hvn_to_hvn_peering": {
+				Description: "Enables automatic HVN to HVN peering when creating a secondary cluster in a federation.",
+				Type:        schema.TypeBool,
+				Optional:    true,
+				ForceNew:    true,
```

Comment:
Maybe you are still working on the PR - but based on the above feedback, add `Computed: true`?